.github/workflows/canister-tests.yml

```diff
@@ -391,6 +390,49 @@ jobs:
         env:
           RUST_BACKTRACE: 1
 
+  test-canisters-script:
+    needs: [cached-build, docker-build-archive]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
+        id: cache
+        with:
+          path: |
+            ~/.cargo
+            target
+          key: ${{ runner.os }}-test-canisters-script-${{ hashFiles('rust-toolchain.toml', 'Cargo.lock', '.node-version', 'package-lock.json') }}
+
+      - name: "Download II wasm"
+        uses: actions/download-artifact@v4
+        with:
+          name: internet_identity_test_cached.wasm.gz
+          path: .
+
+      - name: "Download archive wasm"
+        uses: actions/download-artifact@v4
+        with:
+          name: archive.wasm.gz
+          path: .
+
+      - name: Move wasms
+        run: mv internet_identity_test.wasm.gz internet_identity.wasm.gz
+
+      - name: Create dummy assets
+        run: |
+          mkdir dist
+          touch dist/index.html
+
+      - name: Run tests
+        run: |
+          ./scripts/test-canisters.sh --no-build should_keep_new_anchor_across_rollback
```

Comment:
Maybe add a comment on how you chose these particular tests?

Reply:
done